pipelineruns/notebooks/.tekton/odh-workbench-jupyter-trustyai-cpu-py312-pull-request.yaml

```diff
@@ -43,8 +43,10 @@ spec:
     value: true
   - name: build-platforms
     value:
-    - linux-extra-fast/amd64
+    - linux/x86_64
+    - linux/ppc64le
     - linux/s390x
```

Comment:
This on CPU flavors needs to be removed

Reply:
They were added by @Nash-123 in this PR
https://github.com/red-hat-data-services/konflux-central/commit/ba25bc0cca4e40c13d46336a8f24c91eaf95eeb5